decidim-blogs/app/cells/decidim/blogs/post_g_cell.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
```

Comment:
@entantoencuanto is the template missing?

Reply:
No, the idea is inherit the template from `Decidim::CardGCell directly